scribble/app/views/users/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
I suspect the reason that you're having trouble with over-riding defaults here is that you need to have these views in `app/views/devise` instead of `app/views/users`, but I'm happy to pair with you on this at some point if you want.